.gitignore

```diff
@@ -37,6 +37,9 @@ rootfs-conv/*
 deps/*
 out/*
 
+# protobuf files
+/protobuf/*
```

Comment:
should we also have "protobuf/*" without the root slash?

Reply:
iirc, i ran into an issue where a vendored module had a folder called `protobuf`, and it caused all kinds of shenanigans 😅